GitHub.Collectors/Cache/PointCollectorTableEntity.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.CloudMine.Core.Collectors.Cache;
+using Microsoft.CloudMine.Core.Collectors.Utility;
+
+namespace Microsoft.CloudMine.GitHub.Collectors.Cache
+{
+    public class PointCollectorTableEntity : TableEntityWithContext
+    {
+        public string Url { get; set; }
+        public PointCollectorTableEntity(string url)
+        {
+            this.Url = url;
+            string urlHash = HashUtility.ComputeSha256(this.Url);
+            this.PartitionKey = urlHash;
```

Comment:
>this.PartitionKey = urlHash; [](http://example.com/codeflow?start=12&length=28)

I am assuming we do this mainly because Url has some characters that is not permitted to be a partition key? I wonder if this will cost us read-ability and debug-ability.

Can you potentially experiemnt with Url escaping to see whether it makes the URL safe enough to use as partition key? I am just thinking about making the table user-friendlly to explore in the future. Only if this won't take much time.